scripts/gitVersionFile.ts

```diff
@@ -26,6 +26,7 @@ import fs from 'fs';
 import {join} from 'path';
 
 const specialBranches: {[branch: string]: string} = {
+  airbitz: '',
```

Comment:
What is this branch for?

Reply:
The build system assumes everything is inside the edge-react-gui repo, and uses the branch name to decide what we are building. There is no concept of being in a separate repo, and I don't want to refactor everything to add that concept.

Since the branch name acts like they key into all the config files, this repo is going to get a new "airbitz" branch, which will act somewhat like "master" does in the edge-react-gui repo. That will allow us to key all the config files with the unique Airbitz white-label settings. Also note that I edited the Jenkins file to *only* build the "airbitz" branch, so we don't stomp on top of Edge master.

Reply:
I see. So this is a quick way to setup our CD for this repo. The "correct" but more effortful approach would be to configure a whole other Jenkins repo.